plugins/services/src/js/types/ServiceGroup.ts

```diff
@@ -19,6 +19,8 @@ export type ServiceGroup = {
   id: string;
   name: string;
   quota?: null | ServiceGroupQuota;
+  rolesLength: number;
```

Comment:
I think these should both be a part of `ServiceGroupQuota` and the names could be better.

Reply:
> the names could be better.

I agree, but I didn't want them to be too long like `groupAppWithRolesLength`.